ncm-spma/src/main/pan/components/spma/config.pan

```diff
@@ -5,10 +5,10 @@
 
 unique template components/${project.artifactId}/config;
 
-variable CONFIG_MODULES_CONFIG_SUFFIX ?= 'rpm';
+variable SPMA_BACKEND ?= 'yum';
```

Comment:
@ned21 @jrha I think I may have introduced it... Basically the same thing as in the template library. This provides greater flexibility to use a variable that direct assignment to the path and allows to change the schema without impacting the site configuration (it is enough to assign the variable (if necessary after some massaging) to the new schema element. It also allows to define the variable very early in the config without having the `ncm-spma` configuration already defined.